client/components/createAccount/index.jsx

```diff
@@ -89,15 +89,15 @@ class CreateAccount extends Component {
                     <label htmlFor="name">Username</label>
                     <input
                         name="username"
-                        onChange={event => this.handleChange(event)}
+                        onChange={event => this.onChange(event)}
```

Comment:
nit: shouldn't be creating arrows here - will cause rerenders... 

Probably should be `this.onChange = this.onChange.bind(this)` in constructor then just pass `this.onChange`

Reply:
That wasn't part of my changeset, I just renamed event handler methods to match the event handler names. 🤷‍♂